lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -1034,5 +1040,21 @@ export abstract class ClientApplication {
     setLogger(logger: Logger): void {
         this.logger = logger;
     }
+
+    /**
+     * Called by wrapper libraries (Angular & React) to set SKU and Version passed down to telemetry, logger, etc.
+     * @param sku 
+     * @param version 
+     */
+    initializeWrapperLibrary(sku: string, version: string): void {
+        // Validate the SKU passed in is one we expect
+        if (sku.match(/^@azure\/msal\-[a-z]+$/g) && version.match(/^[0-9a-z\.\-]+$/g)) {
```

Comment:
Could this be done without regex?

Reply:
Also, what would be the downside if "bad" skus and versions were provided?

Reply:
Mostly didn't want bad data polluting our telemetry data but also didn't want to open a door to sending unsanitized data to eSTS/Kusto. I'm positive they sanitize the data as well but I think it's good practice to not blindly accept user provided data as-is

Reply:
Updated to use an enum instead.